### PR TITLE
Fix several CLI flag problems

### PR DESCRIPTION
**Pull request by muXxer**

### CODE REVIEW

File: pkg/config/config.go [hunk 1]
```diff
@@ -108,7 +118,7 @@ func PrintConfig(ignoreSettingsAtPrint ...[]string) {
 	fmt.Println(CfgPeers, PeeringConfig.GetStringSlice(CfgPeers))
 	fmt.Println(CfgPeeringMaxPeers, PeeringConfig.GetStringSlice(CfgPeeringMaxPeers))
 	fmt.Println(CfgPeeringAcceptAnyConnection, PeeringConfig.GetStringSlice(CfgPeeringAcceptAnyConnection))
-	parameter.PrintConfig(ProfilesConfig)
+	//parameter.PrintConfig(ProfilesConfig)
```
Comment (alexsporn) [inline pkg/config/config.go]: on purpose?

File: pkg/model/tangle/transaction_storage.go [hunk 2]
```diff
@@ -137,14 +137,24 @@ func (c *CachedTransaction) Release(force ...bool) {
...
+	return tx, nil
 }
 
-func metadataFactory(key []byte) (objectstorage.StorableObject, int, error) {
-	tx := hornet.NewTransactionMetadata(key[:49])
-	return tx, 49, nil
+func metadataFactory(key []byte, data []byte) (objectstorage.StorableObject, error) {
+	txMeta := hornet.NewTransactionMetadata(key[:49])
+
+	if err := txMeta.UnmarshalObjectStorageValue(data); err != nil {
```
Comment (alexsporn) [inline pkg/model/tangle/transaction_storage.go:153]: why keep the function? I thought OS will never call it now, but always use the factory

File: pkg/model/tangle/bundle_storage.go [hunk 3]
```diff
@@ -101,7 +107,7 @@ func (bundle *Bundle) ObjectStorageValue() (data []byte) {
 	return value
 }
 
-func (bundle *Bundle) UnmarshalObjectStorageValue(data []byte) (consumedBytes int, err error) {
+func (bundle *Bundle) UnmarshalObjectStorageValue(data []byte) (err error) {
```
Comment (alexsporn) [inline pkg/model/tangle/bundle_storage.go:110]: why keep this?

Comment (muXxer) [reply pkg/model/tangle/bundle_storage.go:110]: Less changes in the code before we refactor everything.

File: pkg/model/tangle/transaction_storage.go (hunk 2 above)
Comment (muXxer) [reply pkg/model/tangle/transaction_storage.go:153]: Less changes in the code before we refactor everything.

**Review by alexsporn — APPROVED** [review]

**Pull request merged**